insights/combiners/satellite_version.py

```diff
@@ -2,20 +2,34 @@
 Satellite Version
 =================
 
-Combiner for Satellite Version information. It uses the results of
-the ``Satellite6Version`` parser and the ``InstalledRpms`` parser to determine
-the satellite version
+The following modules are included:
+
+SatelliteVersion - Version of Satellite Server
+----------------------------------------------
+
+CapsuleVersion - Version of Satellite Capsule (>=6.2)
+-----------------------------------------------------
+
+Combiners to get Satellite Server and Capsule Version information. It uses the
+results of the :class:`insights.parsers.satellite_version:Satellite6Version`
+parser and the :class:`insights.parsers.installed_rpms.InstalledRpms` parser to
+determine the version.
```

Comment:
here I think we can explain it more clearly. Since only SatelliteVersion uses Satellite6Version.